sources/api/bootstrap-containers/src/main.rs

```diff
@@ -243,7 +242,8 @@ fn parse_mark_bootstrap_args(args: Vec<String>) -> Result<Subcommand> {
     Ok(Subcommand::MarkBootstrap(MarkBootstrapArgs {
         container_id: container_id,
         // Fail if 'mode' is invalid
-        mode: BootstrapContainerMode::try_from(mode.to_string())
+        mode: mode
+            .parse::<BootstrapContainerMode>()
             .context(error::BootstrapContainerModeSnafu)?,
```

Comment:
This stands out as not being unit tested. Can we add a unit test to this to make sure that we aren't changing behavior?

Reply:
If you mean for `.parse::<BootstrapContainerMode>()`, I think the test is located here: https://github.com/bottlerocket-os/bottlerocket/blob/cf7a49a705883bf5377f3a6de746aa8642a0a2ca/sources/models/src/modeled_types/shared.rs#L966-L981

Unless you mean to unit test something else?

Reply:
I mean `fn parse_metadata_toml`

Reply:
We agreed to open an issue for then entire main.rs file since it's missing tests completely.